libs/island-ui/core/src/lib/Checkbox/Checkbox.css.ts

```diff
@@ -58,7 +58,7 @@ export const largeLabel = style({
   padding: `${theme.spacing[3]}px ${theme.spacing[2]}px`,
   '@media': {
     [`screen and (min-width: ${theme.breakpoints.sm}px)`]: {
-      padding: `${theme.spacing[4]}px ${theme.spacing[3]}px`,
+      padding: `${theme.spacing[3]}px ${theme.spacing[3]}px`,
```

Comment:
This change LGTM but normally when changing design elements in the ui-core lib we should get a designer to confirm the changes (Össi)

Reply:
Ah ofc, sounds good, will do 😄

Reply:
Össi has given his approval ❤️